core-plugins/widgets/Joiner-batchjoiner.json

```diff
@@ -45,5 +45,24 @@
       ]
     }
   ],
-  "outputs": []
+  "outputs": [
+    {
+      "name": "schema",
+      "widget-type": "schema",
+      "widget-attributes": {
+        "schema-types": [
+          "boolean",
```

Comment:
are these types needed? Do the logical types (timestamp, date, etc) show up in the UI with this configuration?

Reply:
removed as it's not needed.